src/test/scala/com/amazon/deequ/suggestions/ConstraintSuggestionRunnerTest.scala

```diff
@@ -225,4 +241,52 @@ class ConstraintSuggestionRunnerTest extends WordSpec with Matchers with SparkCo
 
     assert(expectedConstraintSuggestionJson == actualConstraintSuggestionJson)
   }
+
+  private[this] def suggestHasDataTypeConstraintVerifyTest(data: DataFrame): Unit = {
+
+    val constraints = ConstraintSuggestionRunner()
+      .onData(data)
+      .addConstraintRules(Rules.DEFAULT)
+      .run()
+      .constraintSuggestions
+
+    val hasDataTypeConstraint = constraints
+      .toSeq
+      .head
+      ._2
+      .filter { _.codeForConstraint.startsWith(".hasDataType(") }
+      .head
+
+    val verify = ConstraintSuggestionRunnerTest.verificationFnFromConstraintSrc(
+      hasDataTypeConstraint.codeForConstraint
+    )
+
+    assert(verify(data).status == CheckStatus.Success)
+  }
+}
+
+object ConstraintSuggestionRunnerTest {
+
+  case class Item(value: String)
+
+  def verificationFnFromConstraintSrc(constraint: String): DataFrame => VerificationResult = {
```

Comment:
The objective of this test is to use the code for the suggested constraint.

Since the code is a `String`, this function takes that code and plops it into a function that, when compiled, does the verification.

Is there another way to take the suggested constraint's code and use it directly?

Reply:
thank you, got it now.